### PR TITLE
docs: update nvidia container toolkit section

### PR DESCRIPTION
**Pull request by beholdenkey**
The Nvidia container runtime has been replaced with the Nvidia container toolkit. So I updated that. I have some more steps that can be added to help users set up the Nvidia GPU Operator, as that is a challenge in itself. Other than that, I made some syntax improvements. Please let me know what you think.

### CODE REVIEW

File: docs/advanced.md [hunk 1]
```diff
@@ -155,21 +155,23 @@ You can extend the K3s base template instead of copy-pasting the complete stock
   BinaryName = "/usr/bin/custom-container-runtime"
 
 ```
-## NVIDIA Container Runtime Support
 
-K3s will automatically detect and configure the NVIDIA container runtime if it is present when K3s starts.
+## NVIDIA Container Toolkit Support
+
+K3s will automatically detect and configure the NVIDIA container toolkit if it is present when K3s starts.
```
Comment (brandond) [inline docs/advanced.md:161]: This is not correct. We detect the runtime, not the toolkit. As per https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/index.html:
> The toolkit includes a container runtime [library](https://github.com/NVIDIA/libnvidia-container) and utilities to automatically configure containers to leverage NVIDIA GPUs.

The runtime is all that we detect, we don't actually care if you get it from the toolkit or install it standalone

File: docs/advanced.md [hunk 2]
```diff
@@ -155,21 +155,23 @@ You can extend the K3s base template instead of copy-pasting the complete stock
...
-1. Install the nvidia container runtime packages. For example:
-   `apt install -y nvidia-container-runtime cuda-drivers-fabricmanager-515 nvidia-headless-515-server`
-1. Install K3s, or restart it if already installed:
+    <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html>
+2. Install the nvidia container runtime packages. For example:
+   `apt-get install -y nvidia-container-toolkit cuda-drivers-fabricmanager-515 nvidia-headless-515-server`
+3. Install K3s, or restart it if already installed:
     `curl -ksL get.k3s.io | sh -`
-1. Confirm that the nvidia container runtime has been found by k3s:
+4. Confirm that the nvidia container runtime has been found by k3s:
```
Comment (brandond) [inline docs/advanced.md:169]: there appear to be a bunch of unrelated changes to whitespace and the list numbering. These don't affect the actual rendered doc content. Can you trim your changes down to just the link change and package name?

**Review by brandond — CHANGES_REQUESTED** [review]
Thanks for the contribution! I have a few notes.

**Comment by dereknola** [comment]
@beholdenkey You will also need to sign your commits to comply with DCO

**Pull request closed without merging**